package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@opencensus/opencensus-web-base",
+  "version": "0.0.1",
+  "description": "OpenCensus is a toolkit for collecting application performance and behavior data.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "census-instrumentation/opencensus-web",
+  "scripts": {
+    "postinstall": "npm run bootstrap",
+    "compile": "lerna run compile",
+    "test": "lerna run test",
+    "bootstrap": "lerna bootstrap",
+    "bump": "lerna publish",
+    "codecov": "lerna run codecov"
+  },
+  "keywords": [
+    "opencensus",
+    "browser",
+    "tracing",
+    "profiling"
```

Comment:
We should add ```Stats``` and ```Metrics``` here, I am gonna do the same for OC-Node repo.

Reply:
Done.